app/controllers/v0/mdot/supplies_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module V0
+  module MDOT
+    class SuppliesController < ApplicationController
+      before_action { authorize :mdot, :access? }
+
+      def index
+        supplies = client.get_supplies
```

Comment:
Do you know the timeout for this? I only ask because for synchronous calls, we need to be weary of long timeouts that can block requests and ultimately cause us to run out of puma threads.

Reply:
I am unsure at this point, the DLC haven't even created their side of the integration yet, so it may need to be tweaked in a later PR.

Reply:
OK, so long as you're aware. I don't think we have an official policy or anything in place, but I'll shoot from the hip and say that anything beyond the 10s mark is probably cause for moving this API call to an asynchronous one using a sidekiq worker.

Reply:
Yeah I think that is a fair point, I'll definitely keep that in mind once we start integrating and will refactor if necessary.